### PR TITLE
Lvnod 169 create donor model front end

### PR DESCRIPTION
**Pull request by avoytovich**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/browse/LVNOD-XXX)
* [JIRA subtask](https://ssu-jira.softserveinc.com/browse/LVNOD-XXX)

## Code reviewers

- [ ] @yarvanya
- [ ] @yurii-kokhalevych
- [ ] @shyyka

### Second Level Review

- [ ] @lhalam
- [ ] @4asof4ik

## Summary of issue

ToDo

User will be able to choose gift to present and get info of donors

ToDo

## Testing approach

ToDo
## CHECK LIST
- [ ]  СI passed
- [ ]  All reviewers agreed to merge the PR

### CODE REVIEW

File: app/src/containers/Donor/Donor.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import * as getDonor from './donorActions';
+
+export class Donor extends React.Component {
+
+  render() {
+
+    let abilityTopresent = (this.props.donor.is_available == null ||
```
Comment (shyyka) [inline app/src/containers/Donor/Donor.js]: You can extract the parameter is_available from this.props.donor, instead writing twice  this.props.donor

File: package.json [hunk 2]
```diff
@@ -59,6 +59,7 @@
     "bootstrap-sass": "^3.3.7",
     "clean-webpack-plugin": "^0.1.16",
     "css-loader": "^0.28.1",
+    "react-dropzone": "*",
```
Comment (shyyka) [inline package.json]: This package already exists in 76 row

File: app/src/containers/Donor/Donor.js (hunk 1 above)
Comment (avoytovich) [reply app/src/containers/Donor/Donor.js]: Done

File: package.json (hunk 2 above)
Comment (avoytovich) [reply package.json]: Done

**Review by yarvanya — APPROVED** [review]

**Review by shyyka — APPROVED** [review]

**Review by yurii-kokhalevych — APPROVED** [review]

**Pull request merged**